src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -181,6 +187,251 @@ public Collection<MergeReviewConceptVersions> getMergeReviewConflictingConcepts(
 		return conflicts.values();
 	}
 
+	private boolean isEdited(SnomedComponent<?> lhs, SnomedComponent<?> rhs) {
+		boolean effectiveTime = Objects.equals(lhs.getEffectiveTime(), rhs.getEffectiveTime());
+		boolean buildReleaseHash = Objects.equals(lhs.buildReleaseHash(), rhs.buildReleaseHash());
+
+		return !(effectiveTime && buildReleaseHash);
+	}
+
+	private boolean onlyDifferenceIsDescriptionSynonym(Concept sourceVersion, Concept targetVersion) {
```

Comment:
This method was required as to not remove any historical requirements (i.e. if adding two Synonyms to one Concept, no conflict should arise).